src/http/api/views/account.follows.view.ts

```diff
@@ -183,17 +183,22 @@ export class AccountFollowsView {
                 }
             }
         } catch (err) {
-            throw new Error('Error getting follows');
+            return error('error-getting-follows');
         }
 
         if (!page) {
-            throw new Error('Page not found');
+            return error('no-page-found');
         }
 
         const accounts: AccountInfo[] = [];
-        try {
-            for await (const item of page.getItems()) {
-                const actor = (await item.toJsonLd({
+        const iterator = page.getItems()[Symbol.asyncIterator]();
+
+        while (true) {
```

Comment:
Why use `while(true)` over iterating over `AsyncIterable`?

Reply:
The problem I am trying to solve here is that - `page.getItems()` fails for some items and because it's a part of for loop, there is no error handling so this returns 500. But yeah, while(true) wasn't a good way so I changed the approach a bit.  Used `page.itemIds` this will only get the url and won't fail even if the account is not present in activitypub servers, now we fetch the actual object in a try-catch that will handle the failures gracefully.